source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -909,6 +913,29 @@ See the `Load Balancer Specification <../load-balancers/load-balancers.rst#event
        *  The ID of the Connection
        */
       connectionId: int64;
+
+      /**
+       * The time it took to establish the connection.
+       * In accordance with the definition of establishment of a connection
+       * specified by `ConnectionPoolOptions.maxConnecting`,
+       * it is the time elapsed between the `ConnectionCreatedEvent`
+       * emitted by the same checking out and this event.
+       *
+       * Naturally, this duration is not greater than
+       * `ConnectionCheckedOutEvent`/`ConnectionCheckOutFailedEvent.duration`.
```

Comment:
This doesn't make sense for minPoolSize population, because connections created during minPoolSize population are not associated with any checkout events.  Can we update the phrasing here?

Reply:
Thank you! Done.